src/sentry/static/sentry/app/utils/theme.js

```diff
@@ -93,6 +94,9 @@ const theme = {
   },
 };
 
+// Aliases
 theme.textColor = theme.gray5;
+theme.disabled = theme.red;
```

Comment:
These (disabled/enabled) are bad names, looking for suggestions

Reply:
success / fail? That's what's comes to mind for red/green, disabled to me is more like grey or something